backend/src/main/java/com/google/minicrm/utils/UserAuthenticationUtil.java

```diff
@@ -1,8 +1,6 @@
 package com.google.minicrm.utils;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.*;
```

Comment:
nit: ditto avoid using *, it looks like this may be an IDE setting that's automatically changing single imports to * after you have 3+ or some amount of imports from a single package. Maybe try checking for that?

Reply:
Good catch.